fineract-provider/src/main/java/org/apache/fineract/portfolio/accountdetails/service/AccountDetailsReadPlatformServiceJpaRepositoryImpl.java

```diff
@@ -171,6 +172,12 @@ public AccountSummaryCollectionData retrieveGroupAccountDetails(final Long group
         return this.jdbcTemplate.query(sql, rm, new Object[]{groupId , glimAccount});
     }
 
+    @Override
+    public Collection<LoanAccountSummaryData> retrieveGroupActiveLoanAccountSummary(final Long groupId) {
+        final String loanWhereClause = " where l.group_id = ? and l.loan_status_id = 300 ";
```

Comment:
This still doesn't seem to take the code review feedback that @vishwasbabu had provided in #574 into account?

Reply:
From 

> (since you check if client ID is not null in the service method), it might introduce bugs in the future when someone tries to reuse this method. I would suggest returning only loans of type 3.

I removed the conditional statement to check if client ID is null from LoanReadPlatformServiceImpl

Reply:
@vishwasbabu does this LGTY?

Reply:
I would append a clause to the effect of "l.client_id is NULL", this ensures that only group loans are fetched by the query all JLG loans are ignored (loans which are associated with clients and groups and fetched as a part of the retrieveClientActiveLoanAccountSummary method)